scripts/upgrade

```diff
@@ -24,7 +24,7 @@ sudo rm -rf $final_path
 sudo mkdir -p $final_path
 
 # upgrade to the latest rocketchat
-sudo curl -s -L -o $final_path/rocket.chat-latest.gtar "https://rocket.chat/releases/latest/download"
+sudo curl -s -L -o $final_path/rocket.chat-latest.gtar "https://download.rocket.chat/stable"
```

Comment:
Shouldn't we also use the configured version here? It's kind of dangerous to just download the latest version for several reasons:
- no shasum check here
- no idea if dependencies are met
- untested release by us
etc.

That said I think we also need to think about copying changes we make to the install script in here. Like installing new nodejs versions. If you just upgrade you never get these new dependencies, right?

Reply:
that is right. at the moment this change just focuses on getting the build right again. can you open another issue for this please?

Reply:
Opened issue #34